src/models/User.ts

```
@@ -0,0 +1,36 @@
+export default class User extends GenericModel {
+
+    static TABLE_NAME = "members"
+    static DATABASE_MAP: Object = {
+        id:         "uuid",
+        username:   "username",
+        email:      "fk_email",
+        identity:   "fk_identity",
+        avatar:     "avatar",
+    }
+
+    public id:                  String  = ""
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)